bundles/org.eclipse.swt/Eclipse SWT/common/org/eclipse/swt/graphics/ImageGcDrawer.java

```diff
@@ -45,4 +47,13 @@ public interface ImageGcDrawer {
 	default void postProcess(ImageData imageData) {
 	}
 
+	/**
+     * Returns the GC style used when creating the GC instance.
+     * Default implementation returns <code>SWT.NONE</code>.
+     *
+     * @return the GC style constant
+     */
+    default int getGcStyle() {
```

Comment:
Any reason to not call this simply 
```suggestion
    default int getStyle() {
```
like in other places in SWT as well?

Reply:
The reason being, you can **only** set the style for gc with this method. This is the reason I thought to be descriptive with the name.